shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -826,6 +829,84 @@ public AccessibilityNodeProvider getAccessibilityNodeProvider() {
     }
   }
 
+  /**
+   * Prior to Android Q, it's impossible to add real views as descendants of virtual nodes. This
+   * breaks accessibility when an Android view is embedded in a Flutter app.
+   *
+   * <p>This method overrides a @hide method in {@code ViewGroup} to workaround this limitation.
+   * This solution is derivated from Jetpack Compose, and can be found in the Android source code as
```

Comment:
Do you have the link to the jetpack compose workaround code?

Reply:
it's in the PR description